src/coq_elpi_builtins.ml

```diff
@@ -2325,6 +2325,15 @@ denote the same x as before.|};
             Some (c, np + na))))))),
   DocAbove);
 
+  MLCode(Pred("coq.env.query-extra-dep",
```

Comment:
`coq.env` is for accessing the logical environment.
Please move this code just before 
```
-- Datatypes conversions --------------------------------------------
```
maybe in a dedicated section, say `-- Extra Dependency ------------------------`.

Also I'd call it `coq.extra-dep`